### DISCUSSION TITLE
Calendar-specific icon not visible in the event mode

### DISCUSSION THREAD

**Discussion by modo1302** [totaldebug/atomic-calendar-revive#1214] [Support] [answered]
Hi, can someone advise me on how to get the calendar entity icon to be displayed on the event list? The icon is set, however when the calendar name is displayed on the list, the 'mdi:calendar' icon seems to be hard-coded.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/49bc260f-7f68-4b56-8265-4c5b9829822d)
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/15222c73-e2af-4cf2-af32-fcd722dc42d9)
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/ae6e5505-6131-4485-8171-6318aabe1ed2)

**Comment by marksie1988** [comment]
You need to enable the option to show icons. 

`showEventIcon`

**Reply by modo1302** [reply]
Thanks for the quick answer!
I know that option, I played around with it already. The thing is that the icon will be shown with the event title (in front of it), not even remotely near the caledar name, and that doesn't work for me. Do you know of any way (css maybe) to move it?
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/03f9e0c5-fffa-4410-a294-a3eba7746bde)

**Reply by modo1302** [reply, marked as answer]
I was able to achieve the desired efect by some extreme css styling via cardmod.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/38bebd38-f5ba-403f-b129-7f8c8cb62031)

On a side note - It feels counter-intuitive that the calendar icon is hardcoded and the one set-up in the calendar options is shown as an event icon.

Thanks for your advice though, much appreciated!

**Reply by marksie1988** [reply]
Would you be able to share the CSS in case anyone else has the same issue?

**Reply by modo1302** [reply]
Of course, here you go:
`.event-title {
                width: 265px;
                white-space: nowrap;
                overflow: hidden;
                text-overflow: ellipsis;
                color: white;
             }
             .event-icon {
               float: right;
               margin-top: -3px !important;
             }`
The only downside is that the event title has a fixed width adjusted specifically for my devices. I'm not much of a coder so I do not know how to resolve it in any other way.
